youtube_dl/downloader/__init__.py

```diff
@@ -30,14 +32,17 @@ def get_suitable_downloader(info_dict, params={}):
     protocol = determine_protocol(info_dict)
     info_dict['protocol'] = protocol
 
+    if (info_dict.get('start_time') or info_dict.get('end_time')) and FFmpegFD.can_download(info_dict):
```

Comment:
the feature will be disabled for now.
but for a later time it will be better to add something to specify downloaders priority as it will be cases when multiple downloaders support the same protocols https://github.com/rg3/youtube-dl/pull/8611#issuecomment-186361517.